src/config.py

```diff
@@ -0,0 +1 @@
+API_KEY = ""
```

Comment:
I'm surprised to see this; wasn't it already in the repo? 
Independently, shouldn't it be something like this instead:

perspective_api_key = "TODO_FILL_THIS_IN"?

Reply:
It's not in this branch =/
I'll change the content..